packages/components/src/components/tooltip/tooltip--icon.js

```diff
@@ -0,0 +1,30 @@
+import TooltipDefinition from './tooltip--definition';
```

Comment:
My sincere apologies for not coming up with this sooner, but some other approaches to support tooltip icon (without defining an inherited class here) are:

* Find a common data attribute for definition/icon tooltips (e.g. `data-simple-tooltip`) as possibly for class names
* Change `selectorInit` of definition tooltip to `[data-tooltip-definition],[data-tooltip-icon]`